cli/onionshare_cli/onion.py

```diff
@@ -33,7 +33,7 @@
 import time
 import traceback
 
-from distutils.version import LooseVersion as Version
+from packaging import version as Version
```

Comment:
```suggestion
from packaging.version import Version
```

your code change is not working, we use the Version class later on, an not the version module.

Reply:
Thanks for the feedback. Addressed and added one more change to this file.